config.json

```diff
@@ -924,7 +924,7 @@
 		{
 			"name": "redis tiny (compose)",
 			"valid_from": "2017-01-01",
-			"plan_guid": "53ca5c56-5474-4d64-9211-fe9aee86d502",
+			"plan_guid": "1356eeeb-7c5d-4d9d-9a04-c035a2c709b3",
```

Comment:
The actual GUID is in the Git history of paas-cf:

```bash
git show b3f8018795c55f0f0bd36ed9e7b48d20bc76d87b \
  config/service-brokers/compose/catalog.json
```

It was `a8574a4b-9c6c-40ea-a0df-e9b7507948c8`, but I don't think the actual value matters anymore.